--- gnome-mount-0.5/gnome-mount.schemas.in~	2006-06-22 04:45:53.000000000 +0200
+++ gnome-mount-0.5/gnome-mount.schemas.in	2006-12-05 21:53:52.423361529 +0100
@@ -6,7 +6,7 @@
         <applyto>/system/storage/default_options/vfat/mount_options</applyto>
         <type>list</type>
         <list_type>string</list_type>
-        <default>[shortname=winnt,uid=]</default>
+        <default>[shortname=lower,uid=]</default>
         <locale name="C">
           <short>Default mount options for vfat fs</short>
           <long>A list of default mount options for volumes formatted with the vfat file system.</long>
