Index: src/gnome-mount.c
===================================================================
RCS file: /cvs/gnome/gnome-mount/src/gnome-mount.c,v
retrieving revision 1.16
diff -u -r1.16 gnome-mount.c
--- src/gnome-mount.c	24 Feb 2006 05:22:47 -0000	1.16
+++ src/gnome-mount.c	25 Feb 2006 00:01:42 -0000
@@ -1689,7 +1689,8 @@
 			
 			goto out;
 
-		} else if (fsusage == LIBHAL_VOLUME_USAGE_MOUNTABLE_FILESYSTEM) {
+		} else if (fsusage == LIBHAL_VOLUME_USAGE_MOUNTABLE_FILESYSTEM ||
+			   fsusage == LIBHAL_VOLUME_USAGE_UNKNOWN) {
 			if (volume_mount (udi, volume, drive))
 				rc = 0;
 		}
